deployment/human-connection/README.md

```diff
@@ -6,21 +6,21 @@ just apply our provided configuration files to your cluster.
 
 ## Configuration
 
-Copy our provided templates:
+Change into the `./deployment` directory and copy our provided templates:
 
 ```bash
 $ cp secrets.template.yaml human-connection/secrets.yaml
 $ cp configmap.template.yaml human-connection/configmap.yaml
 ```
 
-Change the `configmap.yaml` as needed, all variables will be available as
+Change the `configmap.yaml` in the `./deployment/human-connection` directory as needed, all variables will be available as
 environment variables in your deployed kubernetes pods.
 
 If you want to edit secrets, you have to `base64` encode them. See [kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/secret/#creating-a-secret-manually).
 
 ```bash
 # example how to base64 a string:
-$ echo -n 'admin' | base64 --wrap 0
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-10T13:50:05Z" title="Friday, May 10th 2019, 3:50:05 pm +02:00">May 10, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -6,21 +6,21 @@ just apply our provided configuration files to your cluster.
 
 ## Configuration
 
-Copy our provided templates:
+Change into the `./deployment` directory and copy our provided templates:
 
 ```bash
 $ cp secrets.template.yaml human-connection/secrets.yaml
 $ cp configmap.template.yaml human-connection/configmap.yaml
 ```
 
-Change the `configmap.yaml` as needed, all variables will be available as
+Change the `configmap.yaml` in the `./deployment/human-connection` directory as needed, all variables will be available as
 environment variables in your deployed kubernetes pods.
 
 If you want to edit secrets, you have to `base64` encode them. See [kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/secret/#creating-a-secret-manually).
 
 ```bash
 # example how to base64 a string:
-$ echo -n 'admin' | base64 --wrap 0
```


@daumie well, OK. It might be unnecessary for short strings, but `base64` will wrap long strings automatically. This might be undesired for certain use case, when a newline would break the integrity of, say, the content of a file.

Reply:
> [<img alt="daumie" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/daumie) **Authored by [daumie](https://github.com/daumie)**
_<time datetime="2019-05-14T10:09:01Z" title="Tuesday, May 14th 2019, 12:09:01 pm +02:00">May 14, 2019</time>_

---


@roschaefer I tested this on Ubuntu 18.04 and it works. 
However, the option `wrap` is not available on MacOS.
Maybe we could be more explicit by specifying that this works on Ubuntu/Linux as opposed to MacOs.